packages/main/src/components/AnalyticalTable/virtualization/VirtualTableBody.tsx

```diff
@@ -2,6 +2,7 @@ import '@ui5/webcomponents-icons/dist/icons/navigation-down-arrow';
 import '@ui5/webcomponents-icons/dist/icons/navigation-right-arrow';
 import React, { useCallback, useEffect, useMemo, useRef } from 'react';
 import { FixedSizeList } from 'react-window';
+import { TableSelectionMode } from '../../../lib/TableSelectionMode';
```

Comment:
should we import from `@ui5/webcomponents-react/lib/TableSelectionMode`?

Reply:
fixed